src/modules/Common/components/Card.tsx

```diff
@@ -34,6 +42,8 @@ const Card: React.FC<CardProps> = ({
         link ? s.card__isLink : null,
         center ? s.card__center : null,
         small ? s.card__isSmall : null,
+        big ? s.card__isBig : null,
+        white ? s.card__isWhite : null,
         className
```

Comment:
Classname can be used with objects

```
classNames(
  s.card,
  {
    link: s.card__isLink,
    center: s.card__center,
  ...
  }
```

Although the notation might seem a bit weird, in the first place, it's actually easier to understand IMO.

Reply:
https://github.com/ambanum/OpenTermsArchive.org/pull/161/commits/d365e6ea8b61443d06072efb16209040bd3a557a